src/Uno.UI.WpfHost/Uno.UI.WpfHost.csproj

```diff
@@ -96,7 +96,7 @@
       <Version>67.0.0</Version>
     </PackageReference>
     <PackageReference Include="System.Reactive">
-      <Version>4.1.0</Version>
```

Comment:
Merging this pull-request would force consumers of the NuGet package `Uno.UI.WpfHost` to upgrade to v4.1.5 as their minver. I don't see any compelling reason to do this at this stage. Thoughts?

Reply:
Indeed, it's not needed. We can skip that update.